core/services/scan.go

```diff
@@ -234,6 +236,13 @@ func (s *ScanService) ScanCVE(ctx context.Context) error {
 				logger.L().Ctx(ctx).Warning("error storing CVE summary", helpers.Error(err),
 					helpers.String("imageSlug", workload.ImageSlug))
 			}
+			if s.vexGeneration {
```

Comment:
@matthyx Where will you have this logic take place?

Reply:
It is already done:
https://github.com/kubescape/helm-charts/pull/309